packages/libbullet/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/bulletphysics/bullet3"
+TERMUX_PKG_DESCRIPTION="SKD for real-time collision detection and multi-physics simulation for robotics, VR, etc"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="ZLIB"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="3.24"
+TERMUX_PKG_SRCURL="https://github.com/bulletphysics/bullet3/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=6b1e987d6f8156fa8a6468652f4eaad17b3e11252c9870359e5bca693e35780b
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS='
+-DBUILD_SHARED_LIBS=ON
+-DINSTALL_LIBS=ON
+-DBUILD_BULLET3=ON
+-DUSE_DOUBLE_PRECISION=OFF
+-DBUILD_PYBULLET=OFF
+-DBUILD_BULLET2_DEMOS=OFF
+-DBUILD_CPU_DEMOS=OFF
+-DUSE_GLUT=OFF
+-DBT_USE_EGL=OFF
+-DUSE_GRAPHICAL_BENCHMARK=OFF
+-DBUILD_EXTRAS=OFF
+-DBUILD_UNIT_TESTS=OFF
+-DCMAKE_BUILD_TYPE=Release
```

Comment:
This was already set in https://github.com/termux/termux-packages/blob/master/scripts/build/configure/termux_step_configure_cmake.sh#L35